numpydoc/validate.py

```diff
@@ -637,7 +637,23 @@ def validate(obj_name, validator_cls=None, **validator_kwargs):
 
     errs = []
     if not doc.raw_doc:
-        if "GL08" not in ignore_validation_comments:
+        report_GL08: bool = True
+        # Check if the object is a class and has a docstring in the constructor
+        if doc.name.endswith("__init__") and doc.is_function_or_method:
```

Comment:
Seems like this is a tiny bit safer
```suggestion
        if doc.name.endswith(".__init__") and doc.is_function_or_method:
```